### PR TITLE
Context menu anywhere on rules

### LINKED ISSUE #555: Be able to remove row when clicking on leftmost column
__Is your feature request related to a problem? Please describe.__

DMN decision table columns can be removed by right-clicking in the column header.
The same is not possible for rows. They can only be removed by clicking into a respective cell. 

__Describe the solution you'd like__

The experience should be consistent for rows and colums.

- [ ] Be able to remove a row via context-menu which is opened via right-clicking into the leftmost column.
- [ ] Be able to remove a column via context-menu which is opened via right-clicking into a cell.

__Describe alternatives you've considered__

Alternatively we could only allow adding / removing rows and columns via the leftmost and top row.

__Additional context__

Enhancement collected during DMN user training.

### PR DESCRIPTION
**Pull request by barmac**
__Which issue does this PR address?__

Closes #555

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: packages/dmn-js-decision-table/src/features/annotations/components/AnnotationCell.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default function AnnotationCell(props) {
   } = row.businessObject;
 
   return (
-    <Cell className="annotation" elementId={ id + '__annotation' }>
```
Comment (pinussilvestrus) [inline packages/dmn-js-decision-table/src/features/annotations/components/AnnotationCell.js:15]: Why we had this suffix in the first place?

Comment (pinussilvestrus) [reply packages/dmn-js-decision-table/src/features/annotations/components/AnnotationCell.js:15]: Kapture 2020-08-07 at 8 36 28

Seems like this is causing unwanted side effects. The annotation content will be displayed as description for some reason.

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/annotations/components/AnnotationCell.js:15]: > Why we had this suffix in the first place?

That's a good question. It does not lead to any element of XML and is not used anywhere.

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/annotations/components/AnnotationCell.js:15]: Fixed :)

**Pull request merged**